doc/modules/ensemble.rst

```diff
@@ -257,10 +257,13 @@ Feature importance evaluation
 The relative rank (i.e. depth) of a feature used as a decision node in a
 tree can be used to assess the relative importance of that feature with
 respect to the predictability of the target variable. Features used at
-the top of the tree contribute to the final prediction decision of a 
-larger fraction of the input samples. The **expected fraction of the 
+the top of the tree contribute to the final prediction decision of a
+larger fraction of the input samples. The **expected fraction of the
 samples** they contribute to can thus be used as an estimate of the
-**relative importance of the features**.
+**relative importance of the features**. In scikit-learn, this fraction
+is combined with the mean decrease in impurity (MDI)
+to create a normalized estimate of improvement in the predictive ability
+of the tree. Refer to [L2014]_ for more information on MDI and feature importance evaluation with Random Forests.
 
 By **averaging** those expected activity rates over several randomized
 trees one can **reduce the variance** of such an estimate and use it
```

Comment:
Maybe add here that this average is referred to as the mean decrease impurity ((MDI) and put the sentence where you cite the reference for more information.